skypy/pipeline/_pipeline.py

```diff
@@ -292,4 +292,4 @@ def get_deps(self, args):
     def __getitem__(self, label):
         name, _, key = label.partition('.')
         item = self.state[name]
-        return item[key] if key else item
+        return item[key].quantity if key else item
```

Comment:
@rrjbca 
I think this runs into problems if the column has a unit type of `None`.

So I think you want something like:
```
        if key:
            return item[key].quantity if item[key].unit else item[key]
        return item
```

Reply:
Thanks, changed this slightly to return the data as numpy array if there are no units (instead of an astropy column)